test/E2ETests/E2EApps/E2EApp/Blob/BlobTriggerBindingFunctions.cs

```diff
@@ -57,7 +58,7 @@ public BlobTriggerBindingFunctions(ILogger<BlobTriggerBindingFunctions> logger)
         {
             var blobStreamReader = new StreamReader(stream);
             string content = await blobStreamReader.ReadToEndAsync();
-            _logger.LogInformation("StreamTriggerOutput: {c}", content);
+            _logger.LogInformation($"StreamTriggerOutput: {content}{Environment.NewLine}");
```

Comment:
Adding newline to deal with failures when newline is missing - [run](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=156587&view=ms.vss-test-web.build-test-results-tab&runId=1360111&resultId=100067&paneView=debug)

```csharp
Assert.Equal() Failure
↓ (pos 11)
Expected: Hello World
Actual:   Hello World[2023-10-24T00:42:04.277Z] Executed 'Func···
↑ (pos 11)
```

[Logs](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=156587&view=ms.vss-test-web.build-test-results-tab&runId=1360111&resultId=100067&paneView=attachments)

```
[2023-10-24T00:42:04.271Z] Executing 'Functions.BlobTriggerStreamTest' (Reason='New blob detected(LogsAndContainerScan): test-trigger-stream-dotnet-isolated/c64a03ba-91fc-4103-a600-10c897da214b.txt', Id=50b2bfa8-f784-4318-ba10-401e799effaf)
[2023-10-24T00:42:04.271Z] Trigger Details: MessageId: 67493b7f-7e77-425e-a8e9-06c07c72e9e1, DequeueCount: 1, InsertedOn: 2023-10-24T00:42:04.000+00:00, BlobCreated: 2023-10-24T00:42:02.000+00:00, BlobLastModified: 2023-10-24T00:42:02.000+00:00
[2023-10-24T00:42:04.276Z] StreamTriggerOutput: Hello World[2023-10-24T00:42:04.277Z] Executed 'Functions.BlobTriggerStreamTest' (Succeeded, Id=50b2bfa8-f784-4318-ba10-401e799effaf, Duration=7ms)
```

Reply:
I am still curious to know what the actual root cause of the test failure. Can we trace back to the code which is emitting `Hello World[2023-10-24T00:42:04.277Z]` ?

Reply:
Discussed offline, but for this issue, unless we believe there's a product bug, the changes should be scoped to the test.

Reply:
My bad, I thought this was a change to the test and not the test app - agree with Fabio that we should not make changes like this